packages/wallet-sdk/src/relay/walletlink/type/Web3Method.ts

```diff
@@ -1,18 +1,21 @@
 // Copyright (c) 2018-2023 Coinbase, Inc. <https://www.coinbase.com/>
 // Licensed under the Apache License, version 2.0
 
-export type Web3Method =
-  | 'requestEthereumAccounts'
-  | 'signEthereumMessage'
-  | 'signEthereumTransaction'
-  | 'submitEthereumTransaction'
-  | 'ethereumAddressFromSignedMessage'
-  | 'scanQRCode'
-  | 'generic'
-  | 'childRequestEthereumAccounts'
-  | 'addEthereumChain'
-  | 'switchEthereumChain'
-  | 'makeEthereumJSONRPCRequest'
-  | 'watchAsset'
-  | 'selectProvider'
-  | 'connectAndSignIn';
+export const web3Methods = [
+  'requestEthereumAccounts',
+  'signEthereumMessage',
+  'signEthereumTransaction',
+  'submitEthereumTransaction',
+  'ethereumAddressFromSignedMessage',
+  'scanQRCode',
+  'generic',
+  'childRequestEthereumAccounts',
+  'addEthereumChain',
+  'switchEthereumChain',
+  'makeEthereumJSONRPCRequest',
+  'watchAsset',
+  'selectProvider',
+  'connectAndSignIn',
+] as const;
```

Comment:
we can derive union type from array, but not other way around. doing this so wallet can check is a string is valid web3method

Reply:
love it